python-qa-tools/utils.py

```diff
@@ -1,61 +1,81 @@
 import argparse, os, json, logging, subprocess
 
-logging.basicConfig(format='%(message)s',
-                    level=logging.DEBUG)
+from stats import check_status
 
-DAEMON = os.getenv('DAEMON')
+logging.basicConfig(format="%(message)s", level=logging.DEBUG)
 
-HOME = os.getenv('HOME')
+DAEMON = os.getenv("DAEMON")
+
+HOME = os.getenv("HOME")
 
 # The function `print_balance_deductions` will print information about the balance deductions after transactions for a wallet or account.
 def print_balance_deductions(wallet, diff):
     if diff > 0:
-        logging.error('Some of the transactions failed')
+        logging.error("Some of the transactions failed")
         logging.info(f"Balance in the {wallet} increased by {diff}")
     elif diff < 0:
-        logging.error('Some of the transactions failed')
+        logging.error("Some of the transactions failed")
         logging.info(f"Balance in the {wallet} decreased by {-1 * diff}")
     else:
-        logging.info(f"All transaction went successfully, No deduction from {wallet} balance")
-        
+        logging.info(
+            f"All transaction went successfully, No deduction from {wallet} balance"
+        )
+
 
 # The utility function `exec_command` is used to execute the cosmos-sdk based commands.
-def exec_command(command):
-    stdout, stderr = subprocess.Popen(command.split(),
-                                    stdout = subprocess.PIPE,
-                                    stderr = subprocess.PIPE).communicate()
-    return stdout.strip().decode(), stderr.strip().decode()
+def exec_command(command, test_type=None, cmd_type=None):
```

Comment:
is there a way to keep it clean. may be we can check for `tx` or `query` substring and get the cmd_type? Similarly how about using an ENV for test_type.